packages/lib/src/components/internal/ClickToPay/context/ClickToPayProvider.tsx

```diff
@@ -49,6 +52,14 @@ const ClickToPayProvider = ({
         ctpService?.subscribeOnStateChange(status => setCtpState(status));
```

Comment:
Do we need any cleanup, to `logout` the shopper before the `ClickToPayProvider` 'destroyed'? Or it's not necessary?

Reply:
That is not necessary. We must logout the shopper only if he clicks in the button to be logged out :)